planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -272,6 +283,27 @@ public static String generateDam(String adp, String monitorGenURL, String monito
         return adpYaml;
     }
 
+    public static Map<String, Object> translateOfferingsType(Map<String, Object> adpYaml) {
+
+        Map<String, Object> topologyTemplate = (Map<String, Object>) adpYaml.get(TOPOLOGY_TEMPLATE);
+        Map<String, Object> nodeTemplates = (Map<String, Object>) topologyTemplate.get(NODE_TEMPLATES);
+
+        for(String moduleName:nodeTemplates.keySet()) {
+            Map<String, Object> module = (Map<String, Object>) nodeTemplates.get(moduleName);
+
+            String moduleType = (String) module.get("type");
+
+            // replaces seaclouds types (unknown by the deployer) with tosca types for the offerings
+            if (moduleType.startsWith("seaclouds.nodes.Platform.")) {
```

Comment:
this looks hacky, better to understand which step is adding `type` with those prefix `seaclouds.nodes.Platform.` and fix that step properly.